code/modules/spells/cantrips.dm

```diff
@@ -0,0 +1,130 @@
+//cantrips: cheap fun spells
+
+//summon food: creates bread or a common fruit at target location
+/spell/aoe_turf/conjure/summon_food
+	name = "Feed The Multitude"
+	desc = "Creates a loaf of bread or a common fruit at the targeted location."
+
+	price = 0.2 * Sp_BASE_PRICE //1 point
```

Comment:
can't use round() in a type definition

can't put it into New() either, because the spellbook uses the initial() proc when building the list of buyable spells